Index: 2.JavaCore/src/com/javarush/task/task18/task1813/AmigoOutputStream.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.javarush.task.task18.task1813;\n\nimport java.io.FileNotFoundException;\nimport java.io.FileOutputStream;\nimport java.io.IOException;\n\n/* \nAmigoOutputStream\n*/\n\npublic class AmigoOutputStream {\n    public static String fileName = \"C:/tmp/result.txt\";\n\n    public static void main(String[] args) throws FileNotFoundException {\n       // new AmigoOutputStream(new FileOutputStream(fileName));\n    }\n\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- 2.JavaCore/src/com/javarush/task/task18/task1813/AmigoOutputStream.java	(revision 2c222a28001943b2ea30773b0ca972c00c4cc4e4)
+++ 2.JavaCore/src/com/javarush/task/task18/task1813/AmigoOutputStream.java	(revision )
@@ -12,7 +12,7 @@
     public static String fileName = "C:/tmp/result.txt";
 
     public static void main(String[] args) throws FileNotFoundException {
-       // new AmigoOutputStream(new FileOutputStream(fileName));
+       //Оз new AmigoOutputStream(new FileOutputStream(fileName));
     }
 
 }
